### DISCUSSION TITLE
목차가 안 보이는데 어떻게 작성해야 자동목차가 뜰까요~?

### DISCUSSION THREAD

**Discussion by HiYongA** [pronist/hELLO#211] [Q&A] [answered]
안녕하세요. 맨 위에 있는 목차와 오른쪽에 가려진 아이콘에 있는 목차가 없는데 어떻게 작성하면 보일까요? 
우선 제 블로그에는
h4
p
p
h4
p
p 태그로 작성이 되어 있어요! 
도움 부탁드립니다!
그리고 스킨 무료로 공유해주셔서 너무 감사드립니다!

**Comment by pronist** [comment, marked as answer]
@HiYongA 안녕하세요.

포스트 내부의 제목의 경우 `h2`, `h3` 태그에만 동작하도록 되어 있으니 참고부탁드립니다.
